### ISSUE TITLE
CoreAnimation engine not rendering shape layer

### ISSUE THREAD

**Issue by snow-google** [airbnb/lottie-ios#1879]
## Which Version of Lottie are you using?

Lottie 3.5.0

Rendering this animation on main thread results in both shape layers being visible. However, rendering using automatic (which presumably uses the core animation renderer) results in the camera shape layer not being rendered. 

## Expected Behavior 
![main thread](https://user-images.githubusercontent.com/121244810/209359262-fd2ef3aa-71da-4131-b1f1-9cc8c98f11a7.png)

## Actual Behavior
![core animation](https://user-images.githubusercontent.com/121244810/209359281-21561633-9ad8-46a4-a1d8-479932bb4ca8.png)

## Animation JSON

{"v":"5.1.10","fr":60,"ip":0,"op":18,"w":96,"h":96,"nm":"Video_Off_On","ddd":0,"assets":[],"layers":[{"ddd":0,"ind":1,"ty":4,"nm":"Video_Icon","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":0,"k":[48,48,0],"ix":2},"a":{"a":0,"k":[12,8,0],"ix":1},"s":{"a":0,"k":[100,100,100],"ix":6}},"ao":0,"hasMask":true,"masksProperties":[{"inv":false,"mode":"s","pt":{"a":1,"k":[{"i":{"x":1,"y":1},"o":{"x":0.4,"y":0},"n":"1_1_0p4_0","t":0,"s":[{"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[2.053,-7.01],[0.363,-5.32],[24.003,18.32],[25.693,16.63]],"c":true}],"e":[{"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[2.053,-7.01],[0.363,-5.32],[0.365,-5.313],[2.055,-7.003]],"c":true}]},{"t":17}],"ix":1},"o":{"a":0,"k":100,"ix":3},"x":{"a":0,"k":0,"ix":4},"nm":"Mask 1"}],"shapes":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":0,"k":{"i":[[0,0],[0,0],[0.74,0],[0,0],[0,-0.73],[0,0],[-0.73,0],[0,0],[0,0.73],[0,0],[0,0],[0,0]],"o":[[0,0],[0,-0.73],[0,0],[-0.73,0],[0,0],[0,0.73],[0,0],[0.74,0],[0,0],[0,0],[0,0],[0,0]],"v":[[18.67,6],[18.67,1.33],[17.33,0],[1.33,0],[0,1.33],[0,14.67],[1.33,16],[17.33,16],[18.67,14.67],[18.67,10],[24,15.33],[24,0.67]],"c":true},"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"fl","c":{"a":0,"k":[0,0,0,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":1,"k":[{"i":{"x":0.999,"y":1},"o":{"x":0.4,"y":0},"n":"0p999_1_0p4_0","t":0,"s":[{"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[0.364,-5.32],[-1.326,-3.63],[22.314,20.01],[24.004,18.32]],"c":true}],"e":[{"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[0.364,-5.32],[-1.326,-3.63],[-1.346,-3.643],[0.344,-5.333]],"c":true}]},{"t":17}],"ix":2},"nm":"Path 2","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"fl","c":{"a":0,"k":[0,0,0,1],"ix":4},"o":{"a":1,"k":[{"i":{"x":[0.833],"y":[0.833]},"o":{"x":[0.167],"y":[0.167]},"n":["0p833_0p833_0p167_0p167"],"t":16,"s":[100],"e":[0]},{"t":17}],"ix":5},"r":1,"nm":"Fill 2","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Group 1","np":2,"cix":2,"ix":3,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":18,"st":0,"bm":0},{"ddd":0,"ind":2,"ty":4,"nm":"BG","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":0,"k":[48,48,0],"ix":2},"a":{"a":0,"k":[0,0,0],"ix":1},"s":{"a":0,"k":[100,100,100],"ix":6}},"ao":0,"shapes":[{"ty":"gr","it":[{"ty":"rc","d":1,"s":{"a":0,"k":[96,96],"ix":2},"p":{"a":0,"k":[0,0],"ix":3},"r":{"a":0,"k":0,"ix":4},"nm":"BG_Path","mn":"ADBE Vector Shape - Rect","hd":false},{"ty":"fl","c":{"a":0,"k":[1,1,1,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Rectangle 1","np":2,"cix":2,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":18,"st":0,"bm":0}],"markers":[]}

**Comment by snow-google** [comment]
I imagine this could be related to an issue fixed last week (https://github.com/airbnb/lottie-ios/issues/1853), but I am unable to check that at this time.

**Comment by nevil** [comment]
@snow-google 

Precondition: I have to set `animationView.contentMode = .center` to get the animation to be visible at all.

I tested the above animation with commit b45fae15. (Commit before the fix for https://github.com/airbnb/lottie-ios/issues/1853)
The issue is reproduced. 

Tested with the latest commit 3b90792 (The fix for https://github.com/airbnb/lottie-ios/issues/1853)
The issue is fixed.


https://user-images.githubusercontent.com/36905/209487951-6895f1b9-5e31-4383-85fd-eb3b0b900c42.mp4
